website/content/en/status/report-2025-04-2025-06/geomman.adoc

```diff
@@ -0,0 +1,26 @@
+=== Geomman Development
+
+Links: +
+link:https://wiki.freebsd.org/SummerOfCode2025Projects/FullDiskAdministrationToolForFreeBSD[Geomman GSoC wiki] +
+link:https://gitlab.com/brauliorivas/geomman[geomman Gitlab repository] +
+link:https://gitlab.com/alfix/bsddialog[bsddialog repository] +
+link:https://man.freebsd.org/cgi/man.cgi?query=sade&manpath=FreeBSD+14.3-RELEASE+and+Ports[sade]
+
+Contact: Braulio Rivas <brauliorivas@FreeBSD.org>
+
+Geomman is a new partition tool based on sade that brings more functionality as moving, copying and pasting partitions. Geomman is part of Google Summer of Code 2025. Currently, it is available on the Gitlab repository. But at some moment, it is expected to be a tool in the base system.
```

Comment:
s/sade/man:sade[8]/
s/as moving/such as moving/
s/copying/copying,/
s/available on the/available in a/
s/moment/future time/
s/to be/to become/